README.md

```diff
@@ -38,7 +38,7 @@ The Pravega Operator manages Pravega clusters deployed to Kubernetes and automat
 ## Requirements
 
 - Kubernetes 1.15+
-- Helm 2.10+
+- Helm 3+
```

Comment:
Does this mean that Helm 2 is not supported anymore?

Reply:
Not officially I don't think; and it likely shouldn't. Helm2 has some significant security concerns (e.g. Tiller)

This article lays down a couple of the issues with Helm2: https://thenewstack.io/helm-3-is-almost-boring-and-thats-a-great-sign-of-maturity/

Do you see any issues with not supporting Helm2 anymore?

Reply:
I asked that because I have a set of instructions to deploy Pravega in Kubernetes and they assume Helm 2, so I will just need to update that. This was the reason for the ask :)